lld/test/ELF/linkerscript/enable-non-contiguous-regions-arm-exidx.test

```diff
@@ -0,0 +1,54 @@
+# If a spilling reorders input sections, the .ARM.exidx table must be rebuilt
```

Comment:
There's also an unwritten convention in LLD to use an extra comment character for non FileCheck commands. For example:
```## If a spilling reorders ...```

It has been some time since that was started so could be worth looking at recent commits to see if it is still in operation.